src/Microsoft.IdentityModel.Tokens/EventBasedLRUCache.cs

```diff
@@ -43,6 +43,8 @@ namespace Microsoft.IdentityModel.Tokens
     /// the map is a <see cref="ConcurrentDictionary{TKey, TValue}"/> which may be modified directly inside an API call or
     /// through eventual processing of the event queue. This implementation relies on the principle of 'eventual consistency':
     /// though the map and it's corresponding linked list may be out of sync at any given point in time, they will eventually line up.
+    /// See here for more details:
+    /// https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/Caching-in-Microsoft.IdentityModel
```

Comment:
This is probably stable, but consider an aka.ms link

Reply:
Good suggestion; added in link: https://aka.ms/identitymodel/caching